src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -308,7 +308,10 @@ private void OnExpanderButtonPreviewMouseMove(object sender, MouseEventArgs e)
                 return;
 
             var senderButton = e.OriginalSource as FrameworkElement;
-            dragDropHelper.HandleMouseMove(senderButton, e.GetPosition(null));
+            var searchElementVM = senderButton.DataContext as NodeSearchElementViewModel;
+            if (searchElementVM != null)
+                dragDropHelper.HandleMouseMove(senderButton, e.GetPosition(null)); 
+
```

Comment:
It'd be nice if you can document why is there a cast to `NodeSearchElementViewModel` but it is not used.